core/src/test/java/org/apache/any23/vocab/RDFSchemaUtilsTest.java

```diff
@@ -43,7 +43,7 @@
      */
     @Test
     public void testSerializeVocabulariesNTriples() {
-        serializeVocabularies(RDFFormat.NTRIPLES, 1918);
+        serializeVocabularies(RDFFormat.NTRIPLES, 2012);//1920
```

Comment:
I have now removed the comment. I was not sure about reason behind this test failure at that time, so added a comment for the future reference, later understood that this is due to changes related to added vocabularies by me.